docker-compose.yml

```diff
@@ -0,0 +1 @@
+docker-compose.prod.yml
```

Comment:
what's the benefit of having this file and `docker-compose.prod.yml` instead of moving the `.prod.yml` file to this?

Reply:
In my experience people seem to not understand the utility of being able to run docker-compose locally.  I did not want to add the prod version as I don't have any way to check that it works.  docker-compose is an extremely useful tool for local development and I think that having a docker-compose.yml that's only for production will be confusing for people that think that docker-compose is only a production tool.

Reply:
Breaking docker-compose.prod.yml out into it's own file, and then symlinking docker-compose.yml makes it easier to tell what each version of the file is for, and makes it easier to watch and maintain changes since they are explicitly named.

Reply:
the problem with a symlink is cross-OS compatibility - regular symlinks work on linux, but tend to fail on windows (never seen a symlink within a git-repository though - maybe it works when checkout is done via windows ...?)